src/firebase/auth/functions/index.js

```diff
@@ -145,7 +150,7 @@ async function checkIfUserExists(data) {
 
 exports.onUserSignUp = functions.auth.user().onCreate(async (user) => {
   if (user.email !== null) {
-    return await produceMessage(user);
+    // return await produceMessage(user);
```

Comment:
Commenting out the Kafka Produce message until kafka is configured @NoahNsimbe

Reply:
Thanks @BenjaminSsempala